src/wallet.rs

```diff
@@ -192,6 +192,10 @@ impl Wallet {
         Ok(enc_sk)
     }
 
+    /// Encrypt the wallet using a password.
+    /// The Utxo cache is dropped.
+    /// The password is salted using a randomly generated salt. The salt is stored in the EncryptedWallet type.
+    /// The EncryptedWallet can be converted back to a Wallet using the EncryptedWallet::decrypt method.
```

Comment:
If you use square-brackets and backticks, rustdoc will generate a link to this function.

```suggestion
    /// The EncryptedWallet can be converted back to a Wallet using the [`EncryptedWallet::decrypt`] method.
```